src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Migration/v0.03/02-setup-procedures.sql

```diff
@@ -0,0 +1,105 @@
+-- Function: delegation.get_all_current_changes
+
+CREATE OR REPLACE FUNCTION delegation.get_all_current_changes(
```

Comment:
Wasn't this supposed to get an updated name specific for offeredby lookup?

Reply:
Forgot to rename it. Did so now.